compiler/il/symbol/OMRResolvedMethodSymbol.cpp

```diff
@@ -579,11 +582,44 @@ OMR::ResolvedMethodSymbol::genInduceOSRCallNode(TR::TreeTop* insertionPoint,
    return insertionPoint->getPrevTreeTop();
    }
 
+/*
+ *  \brief Generate the helper call to request for OSR transition and recompilation of the current compiled method.
```

Comment:
Please document the meaning of the bool return condition for the public APIs you have doxygened in this PR.

Reply:
added documentation to explain the return value. Thanks!